content/verein.de.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Der Verein
+menu:
+  main:
+    parent: project
+    weight: 50
+aliases:
+- /ev
+---
+
+<div style="height: 120px;"></div>
+<div class="container content mb-5">
+
+# OpenBikeSensor -- Verein in Gründung
+
+Im Dezember 2021 haben wir gemeinsam einen Verein gegründet, um uns künftig die
+Arbeit zu erleichern. Der Verein trägt den Namen *OpenBikeSensor* und wird nach
+Eintragung im Vereinsregister Stuttgart den Zusatz "e. V." erhalten.
+
+Die Gründung eines Vereins wurde speziell für die Vereinfachung rechtlicher
+Fragen diskutiert und angestrebt. Dazu zählen hauptsächlich die
+Verantwortlichkeiten für unsere Online-Präsenz und eine gewisse zivilrechtliche
+Haftbarkeit. Eine Auswirkung auf die Weiterentwicklung und den Einsatz der
+Technologie durch die Community soll dies nicht haben.
+
+Mit der erhofften Anerkennung der Gemeinnützigkeit werden sich uns weitere
+finanzielle Möglichkeiten eröffnen. Insbesondere werden wir prüfen, ob und wie
+wir Sammelbestellungen über den Verein abwickeln oder sogar Bausätze verkaufen
+können, um vor allem Einzelpersonen leichter mit Sensoren ausstatten zu können.
+Möglicherweise kann der Verein auch Fördergelder beantragen bzw. erhalten und
+für den Vereinszweck aufwenden.
+
+## Vereinszweck
+
+Der Vereinszweck ist in der Satzung festgelegt und lautet:
+
+> Der Zweck des Vereins ist, unabhängig und parteipolitisch neutral der Unfallprävention im Straßenverkehr, der Förderung der Wissenschaft und Forschung sowie der Volksbildung im Sinne des § 52 Absatz 2 Nr. 1 AO zu dienen.
+>
+> Der Satzungszweck wird insbesondere verwirklicht durch
+> * Förderung und Durchführung wissenschaftlicher Forschungsprojekte zu Themen der Verkehrssicherheit und Verkehrsinfrastruktur, insbesondere bezogen auf nichtmotorisierten Verkehr
+> * Förderung der Entwicklung und Verbreitung von Technologien zur objektiven Messung und Auswertung hierzu relevanter Daten
+> * Öffentliche Bereitstellung gewonnener Daten und Erkenntnisse als Open Data
+> * Vernetzung von Forschenden, öffentlichen Körperschaften und Interessenvertreter:innen zur Förderung der Verwertbarkeit erfasster Daten
+> * Durchführung von öffentlichen Treffen sowie Informations- und Lehrveranstaltungen
+> * Öffentlichkeitsarbeit
+>
+> -- *Quelle: Gründungssatzung des Vereins OpenBikeSensor*
+
+## Mitgliedschaft
+
+Allen natürlichen und juristischen Personen steht es frei, einen Antrag auf
+Aufnahme in den neuen Verein beim Vorstand zu stellen. In Kürze werden wir
+dafür ein Antragsformular und eine Kontaktmöglichkeit hier bereitstellen.
```

Comment:
>In Kürze werden wir dafür ein Antragsformular und eine Kontaktmöglichkeit hier bereitstellen.

Liest sich etwas holprig "hier" zuviel? Oder "In Kürze werden wir hier dafür ein Antragsformular..."?

Reply:
Besser so?

> In Kürze werden wir dafür ein Antragsformular und eine Kontaktmöglichkeit auf dieser Seite bereitstellen.

Reply:
Ja das ist eindeutig!